src/backend/distributed/shared_library_init.c

```diff
@@ -1069,6 +1070,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_propagation_warnings",
+		gettext_noop("Enables citus propagation warnings. It is intended to "
+					 "be used before vanilla tests to stop propagation warnings."),
+		NULL,
+		&EnablePropagationWarnings,
+		true,
+		PGC_USERSET,
+		GUC_SUPERUSER_ONLY | GUC_NO_SHOW_ALL,
```

Comment:
I probably wouldn't mind if a regular user can control this behavior. So, probably not  GUC_SUPERUSER  required?